.azure-pipelines-attestation-container.yml

```diff
@@ -10,14 +10,18 @@ pr:
 trigger:
```

Comment:
I cannot comment on the lines above directly, but it'd be nice to automatically run this pipeline when any files in the `attestation-container` folder has been modified.

Reply:
Fixed in e8e98d8